modules/ROOT/partials/release-notes/deleted.adoc

```diff
@@ -9,13 +9,13 @@ In TinyMCE 5.0, some configurations have been removed because they are no longer
 | Owing to the enhancements to the new inline toolbar behaviour, `fixed_toolbar_container` is not required in TinyMCE 5.0.
 
 | https://www.tiny.cloud/docs-4x/configure/editor-appearance/#insert_button_items[`insert_button_items`]
-| Owing to the changes in the menus and removal of the `context` property, `insert_button_items` is not required in TinyMCE 5.0. Refer to the link:{baseurl}/migration-from-4x/#removedsettings[Migration guide] for a workaround.
+| Owing to the changes in the menus and removal of the `context` property, `insert_button_items` is not required in TinyMCE 5.0. Refer to the xref:migration-from-4x.adoc#removedsettings[Migration guide] for a workaround.
 |===
 
 === Removed themes
 
-* link:{baseurl}/migration-from-4x/#inlite[Inlite]
-* link:{baseurl}/migration-from-4x/#modern[Modern]
+* xref:migration-from-4x.adoc#inlite[Inlite]
+* xref:migration-from-4x.adoc#modern[Modern]
```

Comment:
This xref doesn't resolve as the `modern` id doesn't exist on the relevant header in `migration-from-4x.adoc`. However, it also is broken on production as well. As such this probably needs changing to:
```suggestion
* xref:migration-from-4x.adoc#modernthemeremoved[Modern]
```
and then the id would need to be added to the heading. e.g:
```
[[modernthemeremoved]]
=== Modern theme removed
```

Reply:
done